src/Bracket_60_3_A/sql/world/progression_60_3_a_spell_dbc.sql

```diff
@@ -0,0 +1,2 @@
+#Revoke change Baron Geddon explosion base damage from 3200 to 2100
+DELETE FROM `spell_dbc` WHERE `ID` = 20476;
```

Comment:
```suggestion
DELETE FROM `spell_dbc` WHERE `ID` = 20476;

```

add newline at the end of file